app/views/LicensorViewComponent.vue

```diff
@@ -53,15 +53,63 @@ div.licensor.container(v-else)
         td.border {{prepaid.endDate}}
         td.border {{prepaid.used}} / {{prepaid.maxRedeemers || 0}}
 
-    h3 Create API Client
+    h3 Create/Edit API Client
     form#client-form
         .form-group
             label.small
             | Client name
             =" "
-            input(type="text", name="clientNameCreate")
+            input(type="text", name="clientName")
+        .form-group
+            label.small
+            | minimumLicenseDays
+            =" "
+            input(type="number", value="365", name="minimumLicenseDays")
```

Comment:
Can we pull this default from one place instead of hard-coding it here?

Reply:
Probably true for all these default values we're now setting in the schema and UI.